.vscode/settings.sample.json

```diff
@@ -0,0 +1,5 @@
+{
+  "jest.nodeEnv": {
+    "NODE_ENV": "test"
```

Comment:
Test for react components will fail if value of `NODE_ENV` is not `test`

https://user-images.githubusercontent.com/7823011/188504660-eea794e5-94b7-4b48-95ef-00c9adc9206f.mov